starknet_py/tests/e2e/contract_interaction/v1_interaction_test.py

```diff
@@ -6,6 +6,7 @@
 from starknet_py.tests.e2e.fixtures.contracts import deploy_v1_contract
```

Comment:
> contract_interaction/v1_interaction_test.py fail for cairo 1 contracts compiled with compiler version 1.x. It seems to be a problem on devnet side

Did you verify this on integration node?

Reply:
I haven't tried to run these tests on integration, but the exact error is related to `kind` field in Sierra compiled files: ``Invalid declare transaction v2: missing field `kind`.``
 
Indeed, cairo 1 contracts compiled with compiler v1 do not have this field, but they can be declared using DeclareV2 transaction.

Reply:
Can you please explain what part of starknet.py fails in these tests exactly? Just to make sure the problem is on devnet side?

Reply:
`addDeclareTransaction` fails for Cairo 1 contract compiled with compiler v1 with the error:
```
{'error': {'code': -32602, 'message': 'Invalid declare transaction v2: missing field `kind`'}, 'id': 0, 'jsonrpc': '2.0'}
```
@DelevoXDG  Have you noticed anything similar in jvm?

Reply:
Actually some of those tests pass and it also depends on the specific contract. For some contracts without `kind` field it throws the error and for other it does not :confused: In the current version I added skip condition and reason only for the tests that fail.

Reply:
Is it possible to execute these tests on integration net instead of devnet?

Reply:
> Have you noticed anything similar in jvm?

Not really, declare with cairo 1 contract code seems to work fine on devnet with rpc 0.6.0 (`85495efb71a37ad3921c8986474b7e78a9a9f5fc`) in jvm tests.
Can you send the request the full `addDeclareTransaction ` request that gets sent maybe?